src/main/kotlin/personDsl/api/LanguagesBuilder.kt

```diff
@@ -0,0 +1,5 @@
+package personDsl.api
+
+interface LanguagesBuilder {
+    infix fun String.to(level: Int)
```

Comment:
I think the requirement mentioned using `level` instead of `to` here.
It might be good to check the requirement again just in case!